conf/application.conf

```diff
@@ -52,6 +52,15 @@ controllers {
   }
 }
 
+microservice {
+  services {
+    contact-frontend {
+      host = "localhost"
```

Comment:
Is this maybe going to trigger another config warning? 🤔

Reply:
@ellamdav Hmmm good point...

Reply:
oh yeah, that's a good shout - because, because the microsservices.services stuff isn't auto populated - it has to be set in app-config-base for your service. So we would get a warning unless we set it there, except we only want it locally - so that's why we had the original
```
    platformFrontendHost.getOrElse(
      servicesConfig.getString("contact.frontend.host")
    )
```
where for me at least, the issue is that contact.frontend.host is a badly named, in the sense it's more "urlForContactFrontendWhenRunningLocally"

Reply:
and having it as a key in application.conf means that people running it locally could override it if they wanted, without setting platform.frontend.host which can have other effects since it's used to detect if running remotely by lots of stuff, not that I can think when they would want to override it

Reply:
should we just rename the variable (doesn't have to be what I put, but something more revealing than "contact.frontend.host")

Reply:
🤷‍♂️

Reply:
@oscarduignan Agree with all of the above... This is one of those things that surprisingly complex! I'm now leaning back towards a `local.conf` because really it's only as developers running locally that anyone would ever want to rely on something that's NOT `platform.frontend.host`, I think. 

I did try using the `contact-frontend.host` from `play-frontend-hmrc`'s `reference.conf` but that didn't work.